### ISSUE TITLE
Deprecated features used with Chef: 12.5.1

### ISSUE THREAD

**Issue by tiago-cruz-movile** [chef-cookbooks/powershell#70]
"name":"powershell","version":"3.2.2"

```
Deprecated features used!
  The on_platforms option to node_map has been deprecated at 1 location:
    - /var/chef/cache/cookbooks/powershell/libraries/powershell_module_resource.rb:25:in `<class:PowershellModule>'
```

**Comment by jdgoins** [comment]
+1

**Comment by tas50** [comment]
This should be resolved in the 5.0 release, which I'm just about to push
